integration-tests/browser/yarn.lock

```diff
@@ -63,7 +63,7 @@
     lodash.values "^4.3.0"
 
 "@0x/mesh-browser@./../../browser/":
-  version "8.0.0-beta-0xv3"
+  version "3.0.1-beta"
```

Comment:
This is an auto-generated file and yarn is grabbing the version number from the __package.json__ for the `@0x/mesh-browser` package _locally_. I know this looks weird but it's a side-effect of using a relative import in our __package.json__ instead of importing from npm. I say we leave it be.